docs/current_docs/integrations/openshift.mdx

```diff
@@ -4,9 +4,29 @@ slug: /integrations/openshift
 
 # OpenShift
 
-Dagger provides a Helm chart to create a Dagger Engine DaemonSet on an OpenShift cluster. A DaemonSet ensures that all matching nodes run an instance of Dagger Engine.
+Dagger can be used to set up a Continuous Integration (CI) environment on an OpenShift cluster. This makes it possible to distribute CI workloads across multiple nodes and scale out as needed.
 
-First, create a `values.yaml` file to configure the Dagger Helm deployment. This includes a set of labels for the pod affinity and the taints and tolerations for the nodes.
+## How it works
```

Comment:
@gerhard @matipan @kpenfound would you like to add some more information here? For example:
- How/where does this fit in the typical OpenShift deployment?
- What additional deps if any are needed?
- Are there any limitations or caveats to what has been described here?
- Anything else worth highlighting?